apps/teams-test-app/e2e-test-data/marketplace.json

```diff
@@ -0,0 +1,136 @@
+    },
+    {
+      "title": "AddOrUpdateCartItems API Call - Invalid Price",
+      "type": "callResponse",
+      "boxSelector": "#box_addOrUpdateCartItems",
+      "inputValue": {
+        "cartId": "90dccefe-d23a-497d-9cb7-881bb5aaf729",
+        "cartItems": [
+          {
+            "id": "2",
+            "name": "Item 1",
+            "price": 10,
+            "quantity": 1,
+            "accessories": [{ "id": "11", "name": "Item 11", "price": -5, "quantity": 0 }]
+          }
+        ]
+      },
+      "expectedTestAppValue": "Error: Error: price -5 must be a number not less than 0"
+    },
+    {
+      "title": "AddOrUpdateCartItems API Call - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_addOrUpdateCartItems",
+      "inputValue": {
+        "cartId": "90dccefe-d23a-497d-9cb7-881bb5aaf729",
+        "cartItems": [
+          {
+            "id": "2",
+            "name": "Item 1",
+            "price": 10,
+            "quantity": 1,
+            "accessories": [{ "id": "11", "name": "Item 11", "price": 5, "quantity": 1 }]
+          }
+        ]
+      },
+      "expectedTestAppValue": "{\"id\":\"90dccefe-d23a-497d-9cb7-881bb5aaf729\",\"version\":{\"majorVersion\":1,\"minorVersion\":0},\"cartInfo\":{\"status\":\"Open\",\"market\":\"US\",\"currency\":\"USD\",\"intent\":\"TeamsAdminUser\",\"locale\":\"en-US\",\"createdAt\":\"2021-09-30T15:00:00.000Z\",\"updatedAt\":\" 2021-09-30T15:00:00.000Z\"},\"cartItems\":[{\"id\":\"1\",\"name\":\"Item 1\",\"price\":10,\"quantity\":1,\"accessories\":[{\"id\":\"11\",\"name\":\"Item 11\",\"price\":5,\"quantity\":1,\"imageURL\":\"https://via.placeholder.com/150\"}]},{\"id\":\"2\",\"name\":\"Item 1\",\"price\":10,\"quantity\":1,\"accessories\":[{\"id\":\"11\",\"name\":\"Item 11\",\"price\":5,\"quantity\":1}]}]}"
+    },
+    {
+      "title": "RemoveCartItems API Call - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_removeCartItems",
+      "inputValue": { "cartId": "90dccefe-d23a-497d-9cb7-881bb5aaf729", "cartItemIds": ["1"] },
+      "expectedTestAppValue": "{\"id\":\"90dccefe-d23a-497d-9cb7-881bb5aaf729\",\"version\":{\"majorVersion\":1,\"minorVersion\":0},\"cartInfo\":{\"status\":\"Open\",\"market\":\"US\",\"currency\":\"USD\",\"intent\":\"TeamsAdminUser\",\"locale\":\"en-US\",\"createdAt\":\"2021-09-30T15:00:00.000Z\",\"updatedAt\":\" 2021-09-30T15:00:00.000Z\"},\"cartItems\":[]}"
+    },
+    {
+      "title": "AddOrUpdateCartItems API Call - Invalid ID",
+      "type": "callResponse",
+      "boxSelector": "#box_addOrUpdateCartItems",
+      "inputValue": {
+        "cartId": "90dccefe",
+        "cartItems": [
+          {
+            "id": "2",
+            "name": "Item 1",
+            "price": 10,
+            "quantity": 1,
+            "accessories": [{ "id": "11", "name": "Item 11", "price": 5, "quantity": 1 }]
+          }
+        ]
+      },
+      "expectedTestAppValue": "Error: Error: id must be a valid UUID"
```

Comment:
Not saying this has to be fixed here, but the Error: Error: seems very strange to me. Might be worth creating a work item to go around and fix this.

Reply:
I think I have seen some places for the same prompts when we do negative E2E testing. I am not sure if it is a bug or the way that teams test app handling errors. I'll also tag @erinha for awareness because if we change the way for producing prompts in teams test app, there might be more places need to be updated.

Reply:
Yea you would have to make the changes on the Teams Test App and the E2E Tests as well as this file. It will be a bigger change than the one you are trying to do here, which is why I suggested creating a work item to track this for later. You can assign that to me if you'd like.

Reply:
Thank you. A task has been created. Please check. https://office.visualstudio.com/MetaOS/_workitems/edit/8084720